### PR TITLE
Add spectral flux property

### PR DESCRIPTION
**Pull request by eteq**
See https://github.com/astropy/astropy/pull/10185#pullrequestreview-404383900

### CODE REVIEW

File: astropy/coordinates/spectral_quantity.py [hunk 1]
```diff
@@ -295,3 +295,8 @@ def to(self, unit,
 
     def to_value(self, *args, **kwargs):
         return self.to(*args, **kwargs).value
+
+
```
Comment (astrofrog) [inline astropy/coordinates/spectral_quantity.py:299]: Remove one of the empty lines?

**Comment by astrofrog** [comment]
I think I'm ok with this, can you remove the empty line and then I'll merge

File: astropy/coordinates/spectral_quantity.py [hunk 2]
```diff
@@ -295,3 +295,8 @@ def to(self, unit,
...
+    @property
+    def spectral_density_equivalency(self):
```
Comment (astrofrog) [inline astropy/coordinates/spectral_quantity.py:301]: Can you add a docstring?

**Comment by eteq** [comment]
Will replace this with a PR to astropy since the base PR got merged.

**Pull request closed without merging**